tempCodeRunnerFile.py

```diff
@@ -0,0 +1,57 @@
+import platform
```

Comment:
looks like this is a tmp file that should be removed?

Reply:
+1